tests/test_numdicts/test_numdicts.py

```diff
@@ -0,0 +1,413 @@
+from itertools import product
+import pyClarion.numdicts as nd
+
+import unittest
+import math
+import itertools
+
+from pyClarion.numdicts.numdicts import NumDict
+
+
+class TestNumdicts(unittest.TestCase):
+    # TODO fix/improve look in the iter tools library like product
+    def sequenceGenerator(self, a, b):
```

Comment:
This function is very similar to numpy's `linspace()` function: https://numpy.org/doc/stable/reference/generated/numpy.linspace.html

Using a similar signature might help with cases where we need more than 2 output values. Also, might be worth renaming the result to `linspace()` just so people familiar w/ numpy will have an easier time. If we do end up dividing the tests into multiple test cases, this can just be its own standalone function.